### PR TITLE
{*}[dummy] Hadoop 2.6.0-cdh5.4.5-native + Spark 1.4.1 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@ehiggs: please review?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 7 (6 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/caf25d2c0f5d286cc36d for a full test report.

**Comment by boegel** [comment]
Seems like Hadoop doesn't like getting a newer version of `protobuf`, should be 2.5.0...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6a95c8ba26e07673cd01 for a full test report.

File: easybuild/easyconfigs/h/Hadoop/Hadoop-2.6.0-cdh5.4.5-native.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+patches = ['Hadoop-TeraSort-on-local-filesystem.patch']
+
+builddependencies = [
+    ('Maven', '3.3.3'),
+    ('protobuf', '2.5.0'),  # *must* be this version
+    ('CMake', '3.3.1'),
+    ('snappy', '1.1.3', '', ('GCC', '4.9.3')),
+]
+
+dependencies = [('Java', '1.7.0_80')]
```
Comment (ehiggs) [inline easybuild/easyconfigs/h/Hadoop/Hadoop-2.6.0-cdh5.4.5-native.eb]: Can we bump this?

**Comment by ehiggs** [comment]
Overall, nice. Maybe bump Java to 1.8.

File: easybuild/easyconfigs/h/Hadoop/Hadoop-2.6.0-cdh5.4.5-native.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/h/Hadoop/Hadoop-2.6.0-cdh5.4.5-native.eb]: This is the latest Java 7 release, and it seems like Java 8 + Hadoop is not a good idea yet, see https://issues.apache.org/jira/browse/HADOOP-11090

**Comment by ehiggs** [comment]
There are apparently [issues with Hadoop on Java 8](https://issues.apache.org/jira/browse/HADOOP-11090) so ignore my previous comment.

:shipit:

**Pull request merged**